examples/Inkplate6/Advanced_Inkplate_Features/Inkplate_Wake_Up_On_Touchpads/Inkplate_Wake_Up_On_Touchpads.ino

```diff
@@ -38,21 +39,24 @@ void setup()
     display.begin();
 
     // Setup mcp interrupts
-    display.pinModeInternal(MCP23017_INT_ADDR, display.mcpRegsInt, touchPadPin, INPUT);
     display.setIntOutput(1, false, false, HIGH);
-    display.setIntPin(touchPadPin, RISING);
+    display.setIntPin(PAD1, RISING);
+    display.setIntPin(PAD2, RISING);
+    display.setIntPin(PAD3, RISING);
 
     ++bootCount;
 
     // Our function declared below
     displayInfo();
 
-    // Go to sleep for TIME_TO_SLEEP seconds, but also enable wake up from gpio 34
-    // Gpio 34 is where the mcp interrupt is connected, check
-    // https://github.com/e-radionicacom/Inkplate-6-hardware/blob/master/Schematics%2C%20Gerber%2C%20BOM/Inkplate6%20Schematics.pdf
-    // for more detail
+    // Go to sleep for TIME_TO_SLEEP seconds
     esp_sleep_enable_timer_wakeup(TIME_TO_SLEEP * uS_TO_S_FACTOR);
-    esp_sleep_enable_ext0_wakeup(GPIO_NUM_34, 1);
+
+    // Enable wakeup from deep sleep on gpio 36 (wake button)
+    esp_sleep_enable_ext0_wakeup(GPIO_NUM_36, LOW);
```

Comment:
On my Inkplate6, Pin 36 seems to be always on. I'm not familiar with the hardware specs but I think there isn't a wake button on this model. 
If the ext0 wake is commented out, it works as expected :)

Reply:
You are right. I'll remove this line for the Inkplate6.

Reply:
Fixed!